dist/runCall.js

```diff
@@ -295,6 +301,26 @@ module.exports = function (opts, cb) {
     });
   }
 
+  function computeGetBlockHash(query, cb) {
```

Comment:
let's follow the naming convention of `_callback` for the second argument that we've had up until now.